modules/ossm-release-2-6-1.adoc

```diff
@@ -0,0 +1,37 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/servicemesh-release-notes.adoc
+////
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-release-2-6-1_{context}"]
+= {SMProductName} version 2.6.1
+
+This release of {SMProductName} includes 2.6.1, 2.5.4, and 2.4.10, addresses Common Vulnerabilities and Exposures (CVEs), contains a bug fix, and is supported on {product-title} 4.14 and later.
```

Comment:
With one list item being a list of items, [ISG](https://www.ibm.com/docs/en/ibm-style?topic=punctuation-semicolons#semicolons-between-items-in-a-series) recommends a semicolon for the main list
```suggestion
This release of {SMProductName} includes 2.6.1, 2.5.4, and 2.4.10; addresses Common Vulnerabilities and Exposures (CVEs); contains a bug fix; and is supported on {product-title} 4.14 and later.
```

Alternatively, you could split this into a few sentences that are each less complex